sample_apps/dotnet/README.md

```diff
@@ -10,19 +10,19 @@ Sample .Net application to use Timestream
 
 1. Install required NuGet. Ensure AWSSDK.Core version is 3.3.107 or newer.
    ```shell
-   dotnet add package AWSSDK.Core
-   dotnet add package AWSSDK.TimestreamWrite
-   dotnet add package AWSSDK.TimestreamQuery
+   dotnet add package AWSSDK.Core --version 3.5.1.41
+   dotnet add package AWSSDK.TimestreamWrite --version 3.5.0.20
```

Comment:
Why not use the same versions for both the write and query SDKs

Reply:
The latest version for each of these components are different:
- https://www.nuget.org/packages/AWSSDK.Core/
- https://www.nuget.org/packages/AWSSDK.TimestreamWrite/
- https://www.nuget.org/packages/AWSSDK.TimestreamQuery/